src/App.css

```diff
@@ -1,22 +1,61 @@
-.App {
-  text-align: center;
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  font-size: 22px;
+  font-family: 'PT Sans Narrow', sans-serif;
+
 }
 
-.App-logo {
-  height: 40vmin;
+body {
+  background-color: powderblue;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding: 15vh;
 }
 
-.App-header {
-  background-color: #282c34;
-  min-height: 100vh;
+input {
+  margin-top: 10vh;
+  margin-left: 3vh;
+}
+
+.Router {
   display: flex;
-  flex-direction: column;
+  justify-content: flex-start;
   align-items: center;
-  justify-content: center;
-  font-size: calc(10px + 2vmin);
-  color: white;
+ 
 }
 
-.App-link {
-  color: #09d3ac;
+.Link {
+  display: flex;
+  flex-direction: column;
+  text-decoration: none;
+  color: black;
+  border: 1px solid black;
```

Comment:
no need to change: As a good practice, it's better to use hex values instead of color names.

`black` would be `#000`
`white` would be `#fff`